apps/test-suite/tests/SQLite.ts

```diff
@@ -37,7 +39,7 @@ CREATE TABLE IF NOT EXISTS users (user_id INTEGER PRIMARY KEY NOT NULL, name VAR
     it(`should use newer SQLite version`, async () => {
       const db = await SQLite.openDatabaseAsync(':memory:');
       const row = await db.getFirstAsync<{ 'sqlite_version()': string }>('SELECT sqlite_version()');
-      expect(semver.lte(row['sqlite_version()'], '3.45.3')).toBe(true);
+      expect(semver.gte(row['sqlite_version()'], '3.45.3')).toBe(true);
```

Comment:
original check is incorrect since we are testing "newer" version. we use sqlite 3.47.0 on web (may need to align the version for both native and web later)